common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -415,6 +414,98 @@ public void onReceive(@NonNull PropertyBag propertyBag) {
         return result;
     }
 
+    /**
+     * Performs Account Transfer request with Broker.
+     *
+     * @param parameters a {@link AccountTransferTokenCommandParameters}
+     * @return an {@link AcquireTokenResult}.
+     */
+    @Override
+    public AcquireTokenResult acquireTokenForAccountTransfer(final @NonNull AccountTransferTokenCommandParameters parameters) throws BaseException, ExecutionException, InterruptedException {
+        final String methodTag = TAG + ":acquireTokenForAccountTransfer";
```

Comment:
is there any way to avoid duplicating the code in this method?

Reply:
No longer needed